pennylane/ops/op_math/controlled_ops.py

```diff
@@ -1082,8 +1082,7 @@ def _check_and_convert_control_values(control_values, control_wires):
 
 
 class MultiControlledX(ControlledOp):
-    r"""MultiControlledX(control_wires, wires, control_values, work_wires)
-    Apply a Pauli X gate controlled on an arbitrary computational basis state.
+    r"""Apply a :class:`~.X` gate controlled on an arbitrary computational basis state.
```

Comment:
```suggestion
    r"""Apply a Pauli X gate controlled on an arbitrary computational basis state.
```

This doesn't work for some reason 🤔 ... happy to just leave it be as this^. 

Does this work? 🤔

```suggestion
    r"""Apply a :class:`~.PauliX` gate controlled on an arbitrary computational basis state.
```

Reply:
Second should be working. It's widely used elsewhere